### ISSUE TITLE
<package> sounds misleading? Only use <jsBundleFilePath> instead ?

### ISSUE THREAD

**Issue by qbig** [Microsoft/react-native-code-push#145]
Not sure if this is just me. But in http://microsoft.github.io/code-push/ and several other places like
```
code-push release <appName> <package> <appStoreVersion>
```

the word "package" could be confused with the .apk package file. But it is actually referring to the js bundle file. And I only realised this after I have uploading .apk files for days before I found the app is crashing after every new release and none of the update was applied.
```
01-12 08:07:23.279 14226-14256/com.rnmenu E/AndroidRuntime: FATAL EXCEPTION: mqt_js
                                                            Process: com.rnmenu, PID: 14226
                                                            com.facebook.react.bridge.JSExecutionException: SyntaxError: Invalid character '\u0003' (line 1 in the generated bundle)
                                                                at com.facebook.react.bridge.ReactBridge.loadScriptFromFile(Native Method)
                                                                at com.facebook.react.bridge.JSBundleLoader$1.loadScript(JSBundleLoader.java:34)
                                                                at com.facebook.react.bridge.CatalystInstance$2.run(CatalystInstance.java:138)
                                                                at android.os.Handler.handleCallback(Handler.java:733)
                                                                at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:31)
                                                                at android.os.Looper.loop(Looper.java:136)
                                                                at com.facebook.react.bridge.queue.MessageQueueThread$1.run(MessageQueueThread.java:137)
                                                                at java.lang.Thread.run(Thread.java:841)
```
Maybe it's a good idea to use  'jsBundleFilePath'  consistently? But just want to hear about others' thoughts.

**Comment by lostintangent** [comment]
Hey @gbig! I agree with you that the term "package" is less than ideal. Unfkrtubately, CodePush isn't tied to React Native (it currently supports Cordova, and we're toying with another platform), so we didn't want to call this parameter something that was specific to it. Additionally, even across just React Native apps, the "package" could either be a JS bundle or a directory containing a JS bundle and your assets.

We tried to make the documentation (including within the CLI when you run "code-push release"), but I would really appreciate suggestions for how we could make this clearer.

Additionally, we're planning to add a "release-react" command that lets you do a release that adds React Native specific opinion on top, and so it may make sense for that to be the place that we rename the parameter to something less generic.

**Comment by qbig** [comment]
That sounds great! I did felt puzzled trying to upload apk files while doc mentions that only js and images could be updated. I guess it is specially confusing for people with Android or iOS background

**Comment by lostintangent** [comment]
Yeah I apologize for that confusion. Do you have any thoughts on what we could rename the parameter to? Or do you think it would be sufficient to worry about it within the React specific release command?

**Comment by qbig** [comment]
How about "jsPackage" ? So that it doesn't change much for Cordova users but for RN users it set itself apart clearly from apk or ipa files, which is the major source of confusion.

**Comment by lostintangent** [comment]
What do you think about "updateContents"? That way, it removes the term "package", which can be confusing, but doesn't specifically couple the contents of the update to just JS.

**Comment by qbig** [comment]
Sounds good to me. Definitely better than "package"

**Comment by lostintangent** [comment]
OK cool :) I'll make this change and use this bug to track it. Thanks @qbig!

**Comment by qbig** [comment]
@lostintangent no:) Thanks to you! Really appreciate it.

**Comment by lostintangent** [comment]
I'm going to close this issue since we're tracking this fix as part of the CLI's repo. You can see the PR for this change [here](https://github.com/Microsoft/code-push/pull/110). It will be included in our upcoming 1.5.0 release. Thanks again for providing this awesome feedback!
